README.md

```diff
@@ -41,6 +43,61 @@ fn main() {
 }
 ```
 
+### Generate a new GPX file
+This example only generates tracks. You can add waypoints and routes as well by instantiating new ``Waypoint``s and ``Route``s.
+```rust
+use std::path::Path;
+use gpx::{Gpx, Track, TrackSegment, Waypoint, Route};
+use geo_types::{Point, coord};
+
+pub fn to_gpx<P: AsRef<Path>>(out_path: P) -> Result<String, Box<dyn Error>> {
+    // Instantiate Gpx struct
+    let mut gpx: Gpx = Gpx {
+        version: GpxVersion::Gpx11, // or Gpx10
```

Comment:
I'm personally not a fan of these type annotations / comments here. If you're just skimming the README, it's enough to know you can set e.g. the `waypoints`, but the exact type won't matter too much. And if you've pasted the code into your editor, you'll hopefully get enough info from it.